pkg/kedge/grpc/integration_test.go

```diff
@@ -362,7 +397,23 @@ func (s *BackendPoolIntegrationTestSuite) TestCallToNonSecureBackendLoadBalances
 
 func (s *BackendPoolIntegrationTestSuite) TestCallToUnknownRouteCausesError() {
 	err := grpc.Invoke(s.SimpleCtx(), "/bad.route.doesnt.exist/Method", &unknownResponse{}, &unknownResponse{}, s.proxyConn)
-	require.EqualError(s.T(), err, "rpc error: code = Unimplemented desc = unknown route to service", "no error on simple call")
+	require.EqualError(s.T(), err, "rpc error: code = Unknown desc = failed to proxy request using adhoc router: rpc error: code = Unimplemented desc = unknown route to service", "no error on simple call")
```

Comment:
Again, error should stay the same IMO for this case (and that would consistent with http)